File: FastFood.Web/src/app/core/models/orders/response/orderResponseDto.ts
import { OrderStatusType } from "src/app/core/enums/orderStatusType";
import { OrderProductResponseDto } from "./orderProductResponseDto";

export interface OrderResponseDto{
    id: number;
    orderNumber:string;
    description: string;
    products: OrderProductResponseDto[];
    status: OrderStatusType;
    createdAt: Date;
    updatedAt?: Date;
}

File: FastFood.Domain/Dto/Orders/Response/OrderResponseDto.cs
using FastFood.Domain.Enums;

namespace FastFood.Domain.Dto.Orders.Response
{
    public class OrderResponseDto
    {
        public long Id { get; set; }
        public string OrderNumber { get; set; }
        public string Description { get; set; }
        public List<OrderProductResponseDto> Products { get; set; }
        public OrderStatusType Status { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
    }
}
